modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Numeric/Numeric.tsx

```diff
@@ -238,20 +239,48 @@ const Numeric: React.FC<IProps> = ({
 		}
 
 		if (dataType === 'double') {
+			const newInputMask = {append, appendType, symbols};
+
+			if (typeof numericInputMask === 'string') {
+				const numericInputMaskJSON = JSON.parse(numericInputMask);
+
+				newInputMask.append = numericInputMaskJSON['append'] ?? append;
+
+				newInputMask.appendType =
+					numericInputMaskJSON['appendType'] ?? appendType;
+
+				newInputMask.symbols =
+					numericInputMaskJSON['symbols'] ?? symbols;
+			}
+			else {
+				newInputMask.append = numericInputMask['append'] ?? append;
+
+				newInputMask.appendType =
+					numericInputMask['appendType'] ?? appendType;
+
+				newInputMask.symbols = numericInputMask['symbols'] ?? symbols;
+			}
```

Comment:
I believe it should be rising an exception, since the `numericInputMask` is not in the scope, so you should be geting something like: "Uncaught ReferenceError: numericInputMask is not defined". Just remove the `else` clause and you will be fine, the values you need are already on the `newInputMask`.
```suggestion
```